Show case/add place holder

ryou62525 opened the pull request:
チームのサムネイル部分でバグがあったため修正してリクエストし直しました。

Test: https://ryou62525.github.io/wizcorpsite/

baptistebrunet commented on css/dw-page-modern/css/main.css:
```diff
@@ -655,7 +655,7 @@ textarea {
 }
 
-.row-fluid:after {
+.:after {
```
this looks wrong
.row-fluid was a class selector
. selects nothing

baptistebrunet commented on css/dw-page-modern/style.css:
```diff
@@ -122,7 +122,7 @@ input[type="color"],
 
 .section {
-	background: #fff;
+	background: rgb(255, 255, 255);
```
Why this change ?
is rgb(255, 255, 255) different from #fff ?

baptistebrunet commented on css/slick/slick.css:
```diff
@@ -1,4 +1,4 @@
-/* Slider */
+Slider
```
The comment disappeared 
This doesn't look like a CSS selector

baptistebrunet commented on js/build/scripts.js:
```diff
@@ -884,11 +884,116 @@ jQuery(function($){
+	function setupHtmlGames(){
+
+		var $showcaseSlider = $('#showcase-slider');
+		console.log($showcaseSlider);
```
console.log()

baptistebrunet commented on js/build/scripts.js:
```diff
@@ -884,11 +884,116 @@ jQuery(function($){
+	//team
+	function setupTeam(){
+		var $teamSlider = $('#team-slider');
+		console.log($teamSlider);
```
forgotten console.log()

dyoxyne commented on package.json:
```diff
@@ -21,5 +21,16 @@
     "http-server": "0.9.0",
     "pre-commit": "^1.1.3"
-  }
+  },
```
Unnecessary changes

dyoxyne commented:
日本語版:

![image](https://user-images.githubusercontent.com/2796935/35027818-1bff0d2a-fb96-11e7-8822-43acdcdccc39.png)

変なalignmentです。

ryou62525 commented:
Since the text size increased, the paragraph was lowered, so we changed the text and adjusted the number of characters.

![2018-01-17 16 43 14](https://user-images.githubusercontent.com/12974825/35031409-25587694-fba6-11e7-8586-566bf4d3268d.png)

dyoxyne commented:
- Can you put "autoplay: true" for Portofolio/Team/HTML 5 Games sliders?

- One English mistake: "Samples of the HTML5 game we developed." ==> "Samples of the HTML5 games we developed."

ryou62525 commented:
Added auto play mode and corrected incorrect text.

ronkorving commented on css/dw-page-modern/css/template-responsive.css:
```diff
@@ -1,4 +1,4 @@
-/*! 
+/* ! 
```
I think this space injection needs to be undone.
I think `/*!` maintains the license in minified builds, so best not tampered with.

ronkorving commented on css/dw-page-modern/style.css:
```diff
@@ -155,7 +155,7 @@ input[type="color"],
 	background-image: none;
 	box-shadow: 0 2px 3px rgba(0,0,0,.1);
-	filter: e(%("progid:DXImageTransform.Microsoft.gradient(enabled = false)"));
+	filter: e(("progid:DXImageTransform.Microsoft.gradient(enabled = false)"));
```
What happened here? Is this intentional?
